src/cryptoadvance/specter/rpc.py

```diff
@@ -264,8 +264,29 @@ def multi(self, calls: list, **kwargs):
         url = self.url
         if "wallet" in kwargs:
             url = url+"/wallet/{}".format(kwargs["wallet"])
-        r = requests.post(
-            url, data=json.dumps(payload), headers=headers, timeout=timeout)
+        r = None
+        if 'localhost:' not in url and '127.0.0.1:' not in url:
+            try:
+                requests_session = requests.Session()
+                requests_session = requests.Session()
```

Comment:
```suggestion
                requests_session = requests.Session()
```

Reply:
Good catch, for some reason github doesn't let me accept the suggestion so I'll just close it here and edit manually